DOCUMENTATION.md

```diff
@@ -703,7 +704,7 @@ If the request is not successful, the promise fails with an `err`.
 
 ## Get the status of multiple messages
 
-This API call returns one page of up to 250 messages and statuses. You can get either the most recent messages, or get older messages by specifying a particular notification ID in the [`olderThanId`](#olderthanid) argument.
+This API call returns one page of up to 250 messages and statuses. You can get either the most recent messages, or get older messages by specifying a particular notification ID in the [`olderThanId`](#olderthan-optional) argument.
```

Comment:
It looks like we call the argument `olderThan` rather than `olderThanId`, at least going by the heading this links to.

Reply:
Thanks, that's updated in both places now